frontend/src/screens/internal-apps/AlbyGo.tsx

```diff
@@ -0,0 +1,321 @@
+import { Globe } from "lucide-react";
+import React from "react";
+import { Link } from "react-router-dom";
+import AppHeader from "src/components/AppHeader";
+import ExternalLink from "src/components/ExternalLink";
+import { AppleIcon } from "src/components/icons/Apple";
+import { ChromeIcon } from "src/components/icons/Chrome";
+import { FirefoxIcon } from "src/components/icons/Firefox";
+import { NostrWalletConnectIcon } from "src/components/icons/NostrWalletConnectIcon";
+import { PlayStoreIcon } from "src/components/icons/PlayStore";
+import { ZapStoreIcon } from "src/components/icons/ZapStore";
+import { suggestedApps } from "src/components/SuggestedAppData";
+import {
+  AlertDialog,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+} from "src/components/ui/alert-dialog";
+import { Button } from "src/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "src/components/ui/card";
+import { Input } from "src/components/ui/input";
+import { Label } from "src/components/ui/label";
+import { LoadingButton } from "src/components/ui/loading-button";
+import { useToast } from "src/components/ui/use-toast";
+import { useApp } from "src/hooks/useApp";
+import { createApp } from "src/requests/createApp";
+import { ConnectAppCard } from "src/screens/apps/AppCreated";
+
+export function AlbyGo() {
+  const [loading, setLoading] = React.useState(false);
+  const [appPubkey, setAppPubkey] = React.useState<string>();
+  const [connectionSecret, setConnectionSecret] = React.useState<string>("");
+  const [unlockPassword, setUnlockPassword] = React.useState("");
+  const [showCreateConnectionDialog, setShowCreateConnectionDialog] =
+    React.useState(false);
+  const app = suggestedApps.find((app) => app.id === "alby-go");
+  const { data: createdApp } = useApp(appPubkey, true);
+  const { toast } = useToast();
+  if (!app) {
+    return null;
+  }
+
+  function onClickCreateConnection() {
+    setShowCreateConnectionDialog(true);
+  }
+  async function onSubmitCreateConnection(e: React.FormEvent) {
+    e.preventDefault();
+    setLoading(true);
+    try {
+      const createAppResponse = await createApp({
+        name: "Alby Go",
+        scopes: [
+          "pay_invoice",
+          "get_balance",
+          "get_info",
+          "make_invoice",
+          "lookup_invoice",
+          "list_transactions",
+          "sign_message",
+          "notifications",
+          "superuser",
```

Comment:
Fetch these scopes from `useCapabilities` to avoid errors in other backends

Reply:
thanks, fixed